nevergrad/functions/ml/mlfunctionlib.py

```diff
@@ -25,59 +26,119 @@ class MLTuning(ExperimentFunction):
     """
 
     # Example of ML problem.
-    def _decision_tree_parametrization(self, depth: int, criterion: str, min_samples_split: float, noise_free: bool):
+    def _ml_parametrization(self,
+                            depth: int,  # Parameters for regression trees.
+                            dimension: int,
+                            criterion: str, 
+                            min_samples_split: float,
+                            solver: str,  # Parameters for neural nets.
+                            activation: str,
+                            alpha: float,
+                            learning_rate: str,
+                            regressor: str,  # Choice of learner.
+                            noise_free: bool):
         # 10-folds cross-validation
-        num_data: int = 80
+        num_data: int = 120
         result: float = 0.
         for cv in range(10):
             # All data.
             X_all = np.arange(0., 1., 1. / num_data)
+            random_state = np.random.RandomState(17)
```

Comment:
not sure it's a good idea, maybe providing a seed to the function which is pulled from the random state in the parametrization would be better. This way the function is random and the randomness can be controlled easily through the parametrization